### PR TITLE
[sql-query] support password special chars

### PR DESCRIPTION
**Pull request by maorfr**
by adding quotes when assigning the password.

this is an attempt to fix https://issues.redhat.com/browse/APPSRE-2982

### CODE REVIEW

**Review by apahim — APPROVED** [review]
lgtm, thanks!

File: reconcile/sql_query.py [hunk 1]
```diff
@@ -270,9 +270,12 @@ def make_postgres_command(output, sqlqueries, recipient=None):
...
-            '(time psql',
-            'postgres://$(db.user):$(db.password)@'
-            '$(db.host):$(db.port)/$(db.name)',
+            '(PGPASSWORD="$(db.password)"',
+            'time psql',
+            '--host=$(db.host)',
+            '--port=$(db.port)',
+            '--username=$(db.user)',
+            '--dbname=$(db.name)'
             f'--command "{query}")',
```
Comment (Red-HAP) [inline reconcile/sql_query.py:279]: @maorfr 
You have a risk here with "{query}" as well.

If the query itself contains double-quote chars, you could encounter other bash errors. There should be a more secure way to do this.

Ex:
```sql
select 1 as "cool_column"
  from my_table as "eek";
```

Comment (maorfr) [reply reconcile/sql_query.py:279]: thanks for raising this! this is not an issue, as the query returns escaped from the graphql server. example:
```
      {
        "name": "2021-01-19-compliance-prod-check-profile-policies",
        "query": "SELECT \"version\" FROM \"schema_migrations\" ORDER BY \"version\" DESC LIMIT 10;"
      }
```

Comment (maorfr) [reply reconcile/sql_query.py:279]: thanks for raising this! this is not an issue, as the query returns escaped from the graphql server. example:
```
      {
        "query": "SELECT \"version\" FROM \"schema_migrations\" ORDER BY \"version\" DESC LIMIT 10;"
      }
```

**Pull request merged**